[2022-02-12 20:46:45,709] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.load_officers 2022-02-12T20:45:34.617768+00:00 [queued]>
[2022-02-12 20:46:45,765] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: 1_load_initial_data.load_officers 2022-02-12T20:45:34.617768+00:00 [queued]>
[2022-02-12 20:46:45,767] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 20:46:45,768] {taskinstance.py:881} INFO - Starting attempt 1 of 2
[2022-02-12 20:46:45,770] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2022-02-12 20:46:45,795] {taskinstance.py:901} INFO - Executing <Task(PostgresOperator): load_officers> on 2022-02-12T20:45:34.617768+00:00
[2022-02-12 20:46:45,804] {standard_task_runner.py:54} INFO - Started process 3763 to run task
[2022-02-12 20:46:45,876] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', '1_load_initial_data', 'load_officers', '2022-02-12T20:45:34.617768+00:00', '--job_id', '14', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/load_data.py', '--cfg_path', '/tmp/tmp9_ttafxt']
[2022-02-12 20:46:45,877] {standard_task_runner.py:78} INFO - Job 14: Subtask load_officers
[2022-02-12 20:46:45,999] {logging_mixin.py:112} INFO - Running <TaskInstance: 1_load_initial_data.load_officers 2022-02-12T20:45:34.617768+00:00 [running]> on host f261333f79b3
[2022-02-12 20:46:46,113] {postgres_operator.py:62} INFO - Executing: 
                      COPY dbt_raw_data.officers 
                      FROM '/sample_data/raw__officers.csv' 
                      DELIMITER ',' CSV HEADER;
[2022-02-12 20:46:46,176] {base_hook.py:89} INFO - Using connection to: id: dbt_postgres_instance_raw_data. Host: postgres-dbt, Port: 5432, Schema: dbtdb, Login: dbtuser, Password: XXXXXXXX, extra: None
[2022-02-12 20:46:46,211] {dbapi_hook.py:176} INFO - 
                      COPY dbt_raw_data.officers 
                      FROM '/sample_data/raw__officers.csv' 
                      DELIMITER ',' CSV HEADER;
[2022-02-12 20:46:46,220] {taskinstance.py:1150} ERROR - invalid input syntax for type integer: "2442.0"
CONTEXT:  COPY officers, line 2, column shield_no: "2442.0"
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 984, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/postgres_operator.py", line 65, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 177, in run
    cur.execute(s)
psycopg2.errors.InvalidTextRepresentation: invalid input syntax for type integer: "2442.0"
CONTEXT:  COPY officers, line 2, column shield_no: "2442.0"

[2022-02-12 20:46:46,237] {taskinstance.py:1194} INFO - Marking task as UP_FOR_RETRY. dag_id=1_load_initial_data, task_id=load_officers, execution_date=20220212T204534, start_date=20220212T204645, end_date=20220212T204646
[2022-02-12 20:46:50,548] {local_task_job.py:102} INFO - Task exited with return code 1
